misc/ls-extensions/modules/json-to-record-converter/build.gradle

```diff
@@ -38,6 +38,7 @@ dependencies {
         exclude group: 'com.google.guava', module: 'guava'
     }
     implementation 'com.google.guava:guava:30.0-jre'
+    implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.5'
```

Comment:
We were previously using Jackson-Databind library for json related tasks in the module. Is it possible to replace the gson use cases with that library also? To avoid adding another dependency

Reply:
This is fixed

Reply:
Resolved in https://github.com/ballerina-platform/ballerina-lang/pull/33663/commits/bcf590f93a9af08bae304c79d752b7a8a45e29a0